bddtests/peer_basic.feature

```diff
@@ -10,7 +10,7 @@
 #@chaincodeImagesUpToDate
 Feature: lanching 3 peers
     As a HyperLedger developer
-    I want to be able to launch a 3 peers
+    I want to be able to launch 3 peers
```

Comment:
@ghaskins that's part of the Feature description (just optional text), not part of the DSL. However, kudos for being wary.